src/linux/classes/com/alibaba/wisp/engine/WispTask.java

```diff
@@ -126,6 +126,7 @@ static void trackTask(WispTask task) {
     private String name;
     final Coroutine ctx;                // the low-level coroutine implement
     Status status = Status.ALIVE;
+    boolean shutdownPending;
```

Comment:
assert !shutdownPending in WispTask::set

Reply:
ok